src/components/molecules/Newsletter/Newsletter.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import { useTranslation } from "react-i18next";
+
+import { SubmitButton } from "@atoms/buttons/SubmitButton";
+
+const Newsletter = () => {
+  const { t } = useTranslation();
```

Comment:
We need to see how we can work with the translation with next 13, to not use client components in everyplace